primitives/src/constants.rs

```diff
@@ -9,12 +9,17 @@
 /// ciphersuite identifier for schnorr signature
 pub const CS_ID_SCHNORR: &str = "SCHNORR_WITH_RESCUE_HASH_v01";
 
-/// ciphersuite identifier for BLS signature
-pub const CS_ID_BLS_SIG_NAIVE: &str = "BLS_SIG_WITH_NAIVE_HtG_v01";
+/// ciphersuite identifier for BLS signature, see:
+/// <https://www.ietf.org/archive/id/draft-irtf-cfrg-bls-signature-05.html#name-ciphersuite-format>
+pub const CS_ID_BLS_MIN_SIG: &str = "BLS_SIG_BLS12381G1_XMD:SHA-256_SSWU_RO_NUL_";
```

Comment:
We are definitely not using SSWU hash-to-curve algorithm, but instead a rejection sampling.
I don't know what we should call our current impl, since the identifiers for rejection sampling are not covered by the IRTF draft, but definitely not this.

Reply:
why not? I think my previous notion writeup might confuse you. We are not using our own H2C implementation here, we are just calling `blst::SecretKey::sign()` which internally uses SSWU version of H2C.

Reply:
Ah right. I was confused indeed.